recipe_batches/recipe_batches.py

```diff
@@ -3,7 +3,17 @@
 import math
 
 def recipe_batches(recipe, ingredients):
-  pass 
+    batches = []
+    for ingredient in recipe:
+        if ingredient in ingredients:
+            amount = ingredients[ingredient] / recipe[ingredient]
+            if amount < 1:
+                return 0
+            else:
+                batches.append(math.floor(amount))
+        else:
+            return 0
+    return min(batches)
```

Comment:
Just something to think about in the future here. Instead of using this O(n) operation, there may be a way to use a single integer as the lowest stored batches instead of a list of ingredient. And you just update that integer during every iteration of the for loop.